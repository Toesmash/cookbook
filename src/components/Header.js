import React from 'react';
import { Link } from 'react-router-dom';

const Header = () => (
  <header className='header'>
    <div className='main-container'>
      <div className='header__content'>
        <Link className='header__title' to="/"><h1>CookBook</h1></Link>
        <Link className='header__recipe' to="/create">Add Recipe</Link>
      </div>
    </div>
  </header>
);

export default Header;