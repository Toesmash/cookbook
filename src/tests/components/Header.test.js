import { shallow } from 'enzyme';
import React from 'react';
import Header from '../../components/Header';

test('Header rendering', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});