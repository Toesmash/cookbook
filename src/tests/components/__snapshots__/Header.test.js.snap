// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header rendering 1`] = `
<header
  className="header"
>
  <div
    className="main-container"
  >
    <div
      className="header__content"
    >
      <Link
        className="header__title"
        replace={false}
        to="/"
      >
        <h1>
          CookBook
        </h1>
      </Link>
      <Link
        className="header__recipe"
        replace={false}
        to="/create"
      >
        Add Recipe
      </Link>
    </div>
  </div>
</header>
`;
